api/open_api_3.yaml

```diff
@@ -179,7 +198,7 @@ paths:
         schema:
           type: string
     get:
-      summary: Get the list a specific peer’s datasets
+      summary: Get the list of a specific peer’s datasets
```

Comment:
I didn't end up making any changes to the `/list/:peername` endpoint but am getting the `listing datasets on a peer is not implemented` error when I try to hit the endpoint locally. I tracked down to where the error is triggered in code [here](https://github.com/qri-io/qri/blob/master/lib/datasets.go#L128) but am having trouble following the code path to get there -- am I missing where `UseDscache` would be set to true somewhere ?

Reply:
this is a perfect example of where we've dropped support for a concept along the way, but haven't bothered to update the documentation of it. currently `/list/:peername` doesn't work.

Another note while we're here: we're trying to harmonize around `username` instead of `peername`. It'd be great to switch any reference to `peername` in this doc to `username`

Reply:
I think the right thing to do here would be to drop the documentation of this endpoint entirely. We can re-add it once it's a thing we can do again

Reply:
oh great to know, ok will do. Thanks!